### PR TITLE
add new prereqs to MR

### PR DESCRIPTION
**Pull request by yvonne-liu**
Add new image carousel challenge to prereqs folder in coursebook, to be linked from website.

### CODE REVIEW

File: coursebook/prerequisites/image-carousel.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Build a carousel
+
+![](https://i.imgur.com/zX9HfB0.gif)
```
Comment (oliverjam) [inline coursebook/prerequisites/image-carousel.md]: I know I did it, but probably best not to host the gif on Imgur. They tend to frown on hotlinking and break after a while.

I just uploaded it to this review so this URL should work:

```suggestion
![](https://user-images.githubusercontent.com/9408641/48860569-29c04800-edb9-11e8-8d80-308ce71ba857.gif)
```

Comment (yvonne-liu) [reply coursebook/prerequisites/image-carousel.md]: Thank you ✨

**Comment by eliascodes** [comment]
Merging so this can be linked to from the site. Soz @m4v15

**Comment by m4v15** [comment]
lol

**Comment by m4v15** [comment]
although the change @oliverjam suggested didn't appear to happen?

**Comment by yvonne-liu** [comment]
@m4v15 I fixed it 🤐

**Pull request merged**